Xamarin.Forms.Core/BindableNativeView.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms
+{
+	public abstract class BindableNativeView : View, INativeViewBindableController
```

Comment:
This class is the only one implementing the interface, do we need the interface at all ?